pkg/service/restore_memory.go

```diff
@@ -69,29 +83,51 @@ func (r *RestoreMemory) RestoreByTime(request *model.RestoreTimestampRequest) (i
 	return jobID, nil
 }
 
-func (r *RestoreMemory) restoreByTimeSync(ctx context.Context,
+func (r *RestoreMemory) restoreByTimeSync(
+	ctx context.Context,
 	backend BackupListReader,
 	request *model.RestoreTimestampRequest,
 	jobID int,
 	fullBackups []model.BackupDetails,
 ) {
+	client, aerr := aerospike.NewClientWithPolicyAndHost(
+		request.DestinationCuster.ASClientPolicy(),
+		request.DestinationCuster.ASClientHosts()...)
+	if aerr != nil {
+		err := fmt.Errorf("failed to connect to aerospike cluster, %w", aerr)
+		slog.Error("Failed to restore by timestamp", "routine", request.Routine, "err", err)
```

Comment:
The log entry is different from the one written in `Restore`. Is it possible to extract this client initialization block into a method?

Reply:
good point.